app.json

```diff
@@ -8,7 +8,7 @@
       "ios",
       "android"
     ],
-    "version": "0.2.1",
+    "version": "0.2.2",
```

Comment:
More of a question here, but is it advantageous/necessary to increment the version number outside of releases?

Reply:
Not really, I had been doing it out of habit.

Reply:
I don't have a problem with it, just wasn't sure. Is it something we want to start doing officially?

Reply:
Doesn't matter to me, I had just been using it as a way to create a new upstream branch and draft PR based on the increment commit. I wouldn't say it's necessary, but I'm open to anything.